### PR TITLE
Jds 10/challenge instructions clearer

### LINKED ISSUE #10: Make daily challenge/take home instructions clearer
**Description**
Directions on daily challenges and take homes could be made to be more clear. We could number the instructions so students can more linearly follow directions. Looking at a wall of text can be overwhelming and numbering the directions could help to reduce that feeling of overwhelm.

Going further with the idea of separation we could try and separate instructions into content and GitHub instructions. Maybe have a standard so students know what to expect on each day. Reducing any challenges on the students could improve retention overall.

**Where is the Content Located?**
[Day 7 daily challenge] has a good example of intimidating instructions.

**Old Content**
This is what the day 7 daily challenge currently looks like:

> Create a new repository named "parallax-practice" (Use your bootstrap-boilerplate template)
> Add a collaborator
> Clone your own repository
> Create a new branch named daily-challenge
> Add any 2 fonts from [Google Fonts](https://fonts.google.com/) (Note: the example uses "Lobster" and "Open Sans")
> Add a navbar and a jumbotron with text
> Here's an example of what we're aiming for: [View Example](https://geekwiseacademy.github.io/virtual-mobile-friendly-websites/img/parallax-practice.jpg)
> Add, commit, and push your changes to your daily-challenge branch
> Open a PR and request review from your collaborator
> Review and approve eachother's PR
> Once approved, merge and delete your daily-challenge branch

**New Content**
We could change it to something like this instead:

> ### Daily Challenge Instructions
> 
> 1. Create a new repository named "parallax-practice" (Use your bootstrap-boilerplate template)
> 2. Clone your own repository
> 3. Create a new branch named daily-challenge
> 4. Add any 2 fonts from [Google Fonts](https://fonts.google.com/) (Note: the example uses "Lobster" and "Open Sans")
> 5. Add a navbar and a jumbotron with text
> 
> ### GitHub Instructions
> 
> 1. Add a collaborator
> 2. Add, commit, and push your changes to your daily-challenge branch
> 3. Open a PR and request review from your collaborator
> 4. Review and approve eachother's PR
> 5. Once approved, merge and delete your daily-challenge branch
> 
> **Example:** [View Example](https://geekwiseacademy.github.io/virtual-mobile-friendly-websites/img/parallax-practice.jpg)

**Note**: I'm suggesting these changes for all daily challenge and take home instructions not just day 7. I think a similar change for websites for beginners could also be beneficial. 

[Day 7 daily challenge]: https://geekwiseacademy.github.io/virtual-mobile-friendly-websites/seven.html

### PR DESCRIPTION
**Pull request by ayojackay**
# Changes

_Unordered to Ordered lists, and some emphasis_
- Lists in sections, Daily and Take Home Challenges, were updated to ordered lists instead of unordered
- Added `code` and `strong` elements to provide emphasis for folder naming, detail, and git branches
- Sections with complex Daily Challenge steps, I added two lists, one for complete instructions, and the other for GitHub

Closes #10 

_Before_
Screen Shot 2022-09-28 at 8 31 29 PM

_After_
Screen Shot 2022-09-28 at 8 31 48 PM

Screen Shot 2022-09-28 at 8 34 07 PM

### CODE REVIEW

File: three.html [hunk 1]
```diff
@@ -117,21 +117,30 @@ <h2>Daily Challenge</h2>
       </aside>
       <main class="col-9">
         <ul class="points">
-          <li>Create a new repository named "bootstrap-boilerplate"</li>
-          <li>Clone your repository</li>
-          <li>Create a new branch called "starter-files"</li>
+          <li>Daily Challenge Instructions</li>
```
Comment (22antonio) [inline three.html]: I think `Daily Challenge Instructions` should have a colon at the end. The same with `Github Instructions` below this. That would be:
Daily Challenge Instructions -> Daily Challenge Instructions:
Github Instructions -> Github Instructions:

File: css/style.css [hunk 2]
```diff
@@ -36,14 +36,40 @@ p {
...
+    color: #fff;
+    display: inline-block;
+    content: counter(step);
+    counter-increment: step;
+    width: 45px;
+    height: 45px;
+    border-radius: 23px;
+    text-align: center;
+    margin-right: 10px;
+}
```
Comment (22antonio) [inline css/style.css]: Not sure if this is really necessary but thought I'd bring it to your attention. This custom number for the list items cause the text to be in line with the number. When text goes to a new line it's under the number not under the text like it normally is. Showcase: 
Custom numbers: 
Screen Shot 2022-10-03 at 5 14 42 PM
Traditional numbers:
Screen Shot 2022-10-03 at 5 15 20 PM

Comment (22antonio) [reply css/style.css]: I also feel like the numbers are a little large in comparison to the text.

File: three.html (hunk 1 above)
Comment (22antonio) [reply three.html]: This happens here and on Day 5, 6 and 7

**Review by 22antonio — COMMENTED** [review]
This is all great work 👍! 

I'm just making a comment because none of the changes are dire. The website could survive without them. However, if you do make changes it might be a good opportunity to add a list to [Day 12](https://geekwiseacademy.github.io/virtual-mobile-friendly-websites/twelve.html)'s presentation instructions. 

I also found an unrelated bug while testing out your changes: #13

File: three.html (hunk 1 above)
Comment (ayojackay) [reply three.html]: Screen Shot 2022-10-04 at 11 13 57 AM

File: css/style.css (hunk 2 above)
Comment (ayojackay) [reply css/style.css]: Screen Shot 2022-10-04 at 11 13 30 AM

**Comment by ayojackay** [comment]
@22antonio, I appreciate your feedback. I went ahead and made changes from your suggestions; however, I have yet to fix the size of the number from the numbered list - a future fix xD

**Review by 22antonio — APPROVED** [review]
Thank you for adding in my suggestions. I think this looks great!

**Pull request merged**